extra/python/rx.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# Description: Simple netmap rx example
+# Author: Faisal T. Abu-Nimeh (abunimeh@)
+
+import netmap
+import select
+
+IF_NAME = 'em0'
+
+d=netmap.NetmapDesc('netmap:' + IF_NAME)
+rxr = d.receive_rings[0] # single ring example
+# use poll
+poller = select.poll()
+poller.register(d.getfd(), select.POLLIN)
+
+while True:
+    ready_list = poller.poll() # wait for data
+    if len(ready_list) == 0:
+        print("Timeout occurred")
+        break;
+
+    if not rxr.empty(): # only process nonempty ring
+        i = rxr.cur
+
+        flen = rxr.slots[i].len
+        if flen > rxr.nr_buf_size: continue # drop large packets
+        fbuf = rxr.slots[i].buf[:flen] # this will fail if flen > nr_buf_size, i.e., nr_buf_size must be >= mtu
```

Comment:
Ok, if this happens then it is a bug, probably due to the fact that the netmap support for the adapter you are using (em?) does not check the EOP (end of packet) flag.
What happens with the slots immediately after the one you drop? are they complete packets or just fragments (associated to the dropped one)?